api/app/fire_behaviour/advisory.py

```diff
@@ -73,10 +73,12 @@ def __init__(self,  # pylint: disable=too-many-arguments
         self.cfb = cfb
         self.flame_length = flame_length
         self.sixty_minute_fire_size = sixty_minute_fire_size
-        self.sixty_minute_fire_size_t = None
         self.thirty_minute_fire_size = thirty_minute_fire_size
         self.critical_hours_hfi_4000 = critical_hours_hfi_4000
         self.critical_hours_hfi_10000 = critical_hours_hfi_10000
+        # hfi vs. hfi_t ?
+        # *_t is calculated using ros_t, which is rate of spread since time of ignition.
```

Comment:
Can we replace `*_t` with `*_time_of_ignition`?

Reply:
I'm trying to stick close to the conventions being used in R and in the papers - partly because I'm blindly implementing the math. So can we? Sure. Should we? No idea?

Reply:
Yeah. I'm in favor of deviating from their conventions in favor of clarity, **_unless_** this is standard notation within the fire behaviour domain that enables us to communicate to that audience.

Reply:
I have no idea. Let's ask @nealmcloughlin for an opinion?

Reply:
This PR is getting too busy, will address in next PR.